dotCMS/src/main/java/com/dotmarketing/cms/myaccount/action/MyAccountAction.java

```diff
@@ -37,7 +38,8 @@
 import com.liferay.portal.model.Address;
 import com.liferay.portal.model.User;
 
-public class MyAccountAction extends DispatchAction {
+@Deprecated
+public class MyAccountAction extends SecureAction {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)